web_app/db/crud.py

```diff
@@ -326,3 +326,40 @@ def open_position(self, position_id: uuid) -> Position | None:
             position.status = Status.OPENED.value
             self.write_to_db(position)
         return position.status
+    
+    def get_unique_users(self) -> list[str]:
+        """
+        Fetches all unique users from the Position table.
+        :return: list of unique user ids
+        """
+        with self.Session() as db :
+            try:
+                # Query for distinct user IDs from the Position table
+                unique_users = db.query(Position.user_id).distinct().all()
```

Comment:
please, use `.count()` method for calculating amount of unique users

Reply:
i'm returning a list containing all unique user id's. Am I to return just the total count?

Reply:
> Am I to return just the total count?
Yes, it's